### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Vitalia888-170**

### CODE REVIEW

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -1,28 +1,111 @@
...
+		<div class="flex items-center justify-center mt-3">
+			<input type="tel" placeholder="+38 (067) 00 00 000" class="w-full lg:w-8/12 h-11 rounded-l-lg py-2.5" />
+			<button class="flex items-center justify-center bg-yellow-400 w-12 h-11 rounded-r-lg border border-slate-400">
+				<svg class="h-8 w-8 text-black" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+					<path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M9 5l7 7-7 7" />
+				</svg>
+			</button>
+		</div>
+	</div>
+</div>
```
Comment (Shabaldas) [inline app/views/static_pages/home.html.erb]: ident

**Review by Shabaldas — APPROVED** [review]

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -1,28 +1,111 @@
...
+				<%= image_tag("home_img/low.png") %>
+					<h4 class="ml-0 lg:ml-4">Низька</h4>
+			</div>
+			<div
+				class="w-1/6 flex-col justify-center items-center flex lg:flex-row lg:justify-start text-2xl text-slate-50 mt-8">
+				<%= image_tag("home_img/middle.png") %>
+					<h4 class="ml-0 lg:ml-4">Середня</h4>
+			</div>
+			<div
+				class="w-1/6 flex-col justify-center items-center flex lg:flex-row lg:justify-start text-2xl text-zinc-500 mt-8">
```
Comment (Shabaldas) [inline app/views/static_pages/home.html.erb]: one line class after div

**Review by Shabaldas — CHANGES_REQUESTED** [review]

File: app/views/shared/_header.html.erb [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 			<div class="w-24">
         <%= link_to home_path do %>
 				  <%= image_tag("partials_img/logo.png") %>
         <% end %>
 			</div>
-			<nav class="w-2/3 hidden md:block text-gray-800 font-medium">
+			<nav class="w-2/3 hidden lg:block text-gray-800 font-medium">
 				<ul
-					class="h-full flex flex-col md:flex-row flex-initial justify-between items-center text-center tracking-wide">
+					class="h-full flex flex-col lg:flex-row flex-initial justify-between items-center text-center tracking-wide">
```
Comment (Shabaldas) [inline app/views/shared/_header.html.erb:11]: one line class after ul

Comment (Shabaldas) [inline app/views/shared/_header.html.erb]: one line class after ul

File: tailwind.config.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ module.exports = {
     screens: {
       sm: '640px',
       md: '768px',
-      lg: '1024px',
+      lg: '1025px',
```
Comment (Shabaldas) [inline tailwind.config.js:17]: Does it must be multiple 8 or 5?

File: app/views/static_pages/home.html.erb [hunk 5]
```diff
@@ -1,28 +1,111 @@
...
+	</div>
+	<div class="relative flex flex-col justify-center items-center">
+		<h4 class="absolute bottom-14 text-center text-zinc-500 uppercase font-medium pb-2 bg-stone-200 z-20">Гортай вниз
+		</h4>
+		<div class="absolute bottom-4 z-10">
+			<%= image_tag("home_img/arrow.png") %>
+		</div>
+	</div>
+	<div class="w-full px-4 lg:px-0 lg:w-2/4 mx-auto py-20 text-zinc-500 text-center">
+		<p>Друк здійснюється спеціальним пристроєм — 3D-принтером, який забезпечує створення фізичного об'єкта шляхом
```
Comment (Shabaldas) [inline app/views/static_pages/home.html.erb]: plz. move Ukrainian text in uk.locale file 
And change this text => t.locale
@AndriiBuhalo will help you

**Review by Shabaldas — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 4 above)
Comment (Vitalia888-170) [reply tailwind.config.js:17]: it must be 1025px for ipad pro viewport

File: app/views/static_pages/home.html.erb (hunk 1 above)
Comment (Vitalia888-170) [reply app/views/static_pages/home.html.erb]: fixed

File: app/views/static_pages/home.html.erb (hunk 2 above)
Comment (Vitalia888-170) [reply app/views/static_pages/home.html.erb]: fixed

File: app/views/shared/_header.html.erb (hunk 3 above)
Comment (Vitalia888-170) [reply app/views/shared/_header.html.erb]: fixed

File: app/views/static_pages/home.html.erb (hunk 5 above)
Comment (Vitalia888-170) [reply app/views/static_pages/home.html.erb]: done

**Review by Shabaldas — APPROVED** [review]

**Pull request merged**